config/co2eq_parameters_all.json

```diff
@@ -1385,344 +1778,424 @@
+            "gas": 0.0008736233784356524,
+            "geothermal": 3.3749433766464316e-09,
+            "hydro": 0.9012881267883609,
+            "hydro discharge": 1.4380633727890444e-05,
+            "nuclear": 0.005358223548450256,
+            "oil": 2.352335533522563e-06,
+            "solar": 3.142988339717203e-05,
+            "unknown": 0.01081568537835173,
+            "wind": 0.08142765998447549
+          }
+        ]
       },
       "NO-NO5": {
-        "_source": "electricityMap, 2021 average",
-        "powerOriginRatios": {
-          "battery discharge": 0.0,
-          "biomass": 0.005095892267561793,
-          "coal": 0.0003511880674961801,
-          "gas": 0.023657217353494627,
-          "geothermal": 6.022481163967234e-07,
-          "hydro": 0.8901778245831707,
-          "hydro discharge": 0.0656951918045038,
-          "nuclear": 0.0011415914739590716,
-          "oil": 1.656751552715183e-05,
-          "solar": 0.0003114875073544286,
-          "unknown": 0.0004978884224728323,
-          "wind": 0.013070146299478594
-        }
+        "powerOriginRatios": [
+          {
+            "_source": "electricityMap, 2021 average",
+            "battery discharge": 0.0,
+            "biomass": 0.005095892267561793,
+            "coal": 0.0003511880674961801,
+            "datetime": "2021-01-01",
+            "gas": 0.023657217353494627,
+            "geothermal": 6.022481163967234e-07,
+            "hydro": 0.8901778245831707,
+            "hydro discharge": 0.0656951918045038,
+            "nuclear": 0.0011415914739590716,
+            "oil": 1.656751552715183e-05,
+            "solar": 0.0003114875073544286,
+            "unknown": 0.0004978884224728323,
+            "wind": 0.013070146299478594
+          }
+        ]
       },
       "NZ-NZN": {
-        "_source": "electricityMap, 2021 average",
-        "powerOriginRatios": {
-          "battery discharge": 0.0,
-          "biomass": 0.0,
-          "coal": 0.10300324286854977,
-          "gas": 0.15085195353697833,
-          "geothermal": 0.30942699443458854,
-          "hydro": 0.31789333594944125,
-          "hydro discharge": 0.0,
-          "nuclear": 0.0,
-          "oil": 0.0007836425136503577,
-          "solar": 0.0,
-          "unknown": 0.04927664985922736,
-          "wind": 0.06878744399718065
-        }
+        "powerOriginRatios": [
+          {
+            "_source": "electricityMap, 2021 average",
+            "battery discharge": 0.0,
+            "biomass": 0.0,
+            "coal": 0.10300324286854977,
+            "datetime": "2021-01-01",
+            "gas": 0.15085195353697833,
+            "geothermal": 0.30942699443458854,
+            "hydro": 0.31789333594944125,
+            "hydro discharge": 0.0,
+            "nuclear": 0.0,
+            "oil": 0.0007836425136503577,
+            "solar": 0.0,
+            "unknown": 0.04927664985922736,
+            "wind": 0.06878744399718065
+          }
+        ]
       },
       "NZ-NZS": {
```

Comment:
How come the script is only creating "NZ-NZS" and not a "NZ", when we have CSVs for both? 🤔 
And can I safely remove the NZS and NZN files here? (Sorry offtopic discussion, figured it made sense when you're already working on the script 😄)
<img width="266" alt="Screenshot 2022-03-18 at 07 22 09" src="https://user-images.githubusercontent.com/3296643/158948668-eddfc385-eed0-4cde-b788-2ffb61fdbda8.png">

Reply:
There is a requirement for having a certain number of timestamps with valid data to generate a yearly aggregate. + the last year we are looking at is 2021. I think this explains why NZ does not get its yearly aggregate. 
For NZ-NZN and NZ-NZS, yes we can move the csvs away from there, I'll do it :)

Reply:
Ah okay, makes sense - thanks! 🙌